### PR TITLE
Add discounts page

### PR DESCRIPTION
**Pull request by ivanaur**
PR based on Sonja's draft.

### CODE REVIEW

**Comment by markoa** [comment]
Does it make sense to title it "Free student accounts"? Same for educators?

I think we want edu people to have free accounts for education purposes, and a discount for their internal R&D. /cc @fbrdaric

**Comment by shiroyasha** [comment]
One nitpick @ivanaur. Please keep the lines up to 80 characters as described here https://github.com/semaphoreci/docs/blob/master/CONTRIBUTING.md#text-formatting.

File: discounts_5cb8501f2c7d3a18ac7ac51f.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Aside from regular $20 free credits per month, students will receive additional $240 gift credits each year, for as long as your studies last.
+
+In order to request a student account, please send us an email to [customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com) from your school-issued email address. In case you don’t have one, please provide some document in order to verify your status of a student.
+
+We will evaluate your request, and get back to you with an answer as soon as possible.
+
+## Educator accounts
+
+Individual educators currently teaching programming at accredited high schools, colleges, or universities — as well as any learn-to-code programs or bootcamps — are eligible for an educator account.
```
Comment (fbrdaric) [inline discounts_5cb8501f2c7d3a18ac7ac51f.md]: @ivanaur can we put back here are eligible for a free account?

File: discounts_5cb8501f2c7d3a18ac7ac51f.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Every student currently enrolled at an accredited high school, college, or university, as well as any learn-to-code program or bootcamp, is eligible for a student account.
+
+Aside from regular $20 free credits per month, students will receive additional $240 gift credits each year, for as long as your studies last.
+
+In order to request a student account, please send us an email to [customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com) from your school-issued email address. In case you don’t have one, please provide some document in order to verify your status of a student.
+
+We will evaluate your request, and get back to you with an answer as soon as possible.
+
+## Educator accounts
```
Comment (fbrdaric) [inline discounts_5cb8501f2c7d3a18ac7ac51f.md]: Can we add the following sentence on providing discounts for supporting edu internal projects:

Researchers or collaborators working on a project within any institution whose purpose directly relates to learning, teaching, and/or training, although may not directly be involved in the teaching process, are eligible for the discount.

Comment (fbrdaric) [reply discounts_5cb8501f2c7d3a18ac7ac51f.md]: Maybe it can be a separate paragraph/topic.

Comment (ivanaur) [reply discounts_5cb8501f2c7d3a18ac7ac51f.md]: Call it **Research discounts** ?

**Pull request merged**